src/SchemaRegistry.ts

```diff
@@ -28,9 +29,10 @@ export default class SchemaRegistry {
   private api: SchemaRegistryAPIClient
   public cache: Cache
 
-  constructor({ auth, clientId, host, retry }: SchemaRegistryAPIClientArgs) {
+  constructor({ auth, clientId, host, retry }: SchemaRegistryAPIClientArgs,
+              forSchemaOptions?: Partial<ForSchemaOptions>) {
```

Comment:
Could this second parameter be an object instead? That way it'd be more easily extendable if we wish to add more `opts` in the future :)

Reply:
could we just add to `SchemaRegistryAPIClientArgs` ?

Reply:
could we just add to SchemaRegistryAPIClientArgs?

Reply:
I wouldn’t want to mix this into the APIClient options since it’s a completely separate thing. 
That’s why I suggested adding a second argument, and to make it future-safe it’d be an object 🙂

Reply:
Agree, just pushed again :-)